coding-guide-helper/src/anti-patterns-examples.ts

```diff
@@ -0,0 +1,20 @@
+// This code uses multiple comparisons in a single if statement
+export const antiPatternExample_01 = (value: string) => {
+  if (value === 'a' || value === 'b' || value === 'c') {
```

Comment:
Suggestion:
- [Prefer `Array.includes()` Over Multiple Comparisons](https://github.com/amwebexpert/chrome-extensions-collection/blob/master/coding-guide-helper/public/markdowns/common-coding-patterns.md#prefer-arrayincludes-over-multiple-comparisons)

Reply:
Suggestion:
- [Prefer Using `??` Over `||` for Handling `null` and `undefined`](https://github.com/amwebexpert/chrome-extensions-collection/blob/master/coding-guide-helper/public/markdowns/common-coding-patterns.md#prefer-using--over--for-handling-null-and-undefined)